src/stores/SSEChannel/MessagesSSEChannel.ts

```diff
@@ -105,21 +105,21 @@ export class MessagesSSEChannel extends SSEChannel<EventMessage> {
 		return messagesChunk;
 	};
 
-	private initConnection = (resumeFromId?: string): void => {
+	private initConnection = (resumeMessageIds?: string[]): void => {
 		this.closeChannel();
 		this.resetSSEState({ isLoading: true });
 		this.clearFetchedChunkSubscription();
 
-		const messageId: string[] = this.messagesIdsEvent
-			? (Object.values(this.messagesIdsEvent.messageIds).filter(Boolean) as string[])
-			: [];
+		const messageId: string[] =
+			resumeMessageIds ??
```

Comment:
Do we need messagesIdsEvent now?
if not maybe we should move ResumeMessageIdsStore inside MessagesSSEChannel ?

Reply:
nvm we need it